docs/092-ht_change_voice.md

```diff
@@ -1,3 +1,54 @@
 # How To Change Your Assistant's Voice
 
-**WIP**
+## Using TTS server plugin
+
+TTS server plugin is using services hosted by OVOS community:
+https://openvoiceos.github.io/community-docs/325-members/#members-hosting-services
+
+That reduces local resource consumption. Its recommended to use TTS server plugin for resource constraint hardware (eg. RPi 3)
+
+Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.
+
+If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config
+
+```json
+{
+    "tts": {
+        "ovos-tts-plugin-server": {
+            "voice": "amy-low"
+        }
+    }
+}
+```
+
+Reload configuration (an example used with systemd services `systemctl --user restart ovos`)
```

Comment:
This and the line below are duplicated and I don't think its necessary.  Maybe just put these in a "next step" after changing the config file?